services/explorer/src/api/websocket/mod.rs

```diff
@@ -394,17 +402,20 @@ impl Server {
     fn handle_transaction(
         &self,
         message: Transaction,
-    ) -> impl Future<Item = TransactionResponse, Error = failure::Error> {
+    ) -> LocalBoxFuture<'static, Result<TransactionResponse, failure::Error>> {
```

Comment:
Nit: I'd use `impl Future` here and move boxing closer to where it's necessary (i.e., `Handler<Transaction>` implementation).